src/test/java/com/wikia/webdriver/common/core/urlbuilder/UrlBuilder.java

```diff
@@ -64,22 +74,33 @@ private String getUrlPrefix(String wikiName) {
   }
 
   private String getUrlSuffix(String wikiName) {
+    if (env.contains("dev") && isMercuryBrowser()) {
+      return String.format(XIPIO_ADDRESS_FORMAT, XIPIO_DEFAULT_DOMAIN, XIPIO_DEFAULT_PORT);
```

Comment:
I would go with:
```java
return ".127.0.0.1.xip.io:8000";
``` 
and get rid of the 3 static vars. 
It would be more readable for me.